src/main/java/ua/okwine/productexpirationdate/entity/Product.java

```diff
@@ -0,0 +1,117 @@
+package ua.okwine.productexpirationdate.entity;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name="product")
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "vendor_code")
+    private String vendorCode;
+
+    @Column(name = "barcode")
+    private String barCode;
+
+    @Column(name = "product_name")
+    private String productName;
+
+    @Column(name = "produced")
+    @DateTimeFormat(pattern = "yyyy-MM-dd")
+    private Date produced;
```

Comment:
Date is an old class which has a lot of downsides. You should use it's replacement from Java 8+ LocalDate